packages/react-native-skia/components/ParagraphTestApp.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { View, AppRegistry, Image, Text } from 'react-native';
```

Comment:
Image can be removed if not used in test code.

Reply:
As Image component is not used in the test app , removed it.